crates/diesel_models/src/payment_intent.rs

```diff
@@ -24,11 +23,7 @@ pub struct PaymentIntent {
     pub description: Option<String>,
     pub return_url: Option<String>,
     pub metadata: Option<serde_json::Value>,
-    pub connector_id: Option<String>,
-    pub shipping_address_id: Option<String>,
-    pub billing_address_id: Option<String>,
     pub statement_descriptor_name: Option<String>,
```

Comment:
This field should be renamed to statement_descriptor but in database we won't rename instead we add new field